wait-for-running-branch.sh

```diff
@@ -39,3 +40,19 @@ until $(echo $STATUS | grep -wqe "Ready\|NeedsPriming" ); do
   ((COUNT++))
   echo "Branch status now = $STATUS https://calypso.live/status?branch=$BRANCHNAME"
 done
+
+SITE=$(curl https://calypso.live/?branch=$BRANCHNAME 2>/dev/null)
+COUNT=0
+until ! $(echo $SITE | grep -q "dserve" ); do
```

Comment:
I think it would be good to wait for a more specific string here.  A couple of options:

- Calypso source contains `<title>WordPress.com</title>`
- Calypso source contains `<div id="wpcom"` (the full div is `<div id="wpcom" class="wpcom-site">` but I assume the latter part could change)
- dserve source contains `<div class="dserve-message">`